srcpkgs/wally-cli/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'wally-cli'
+pkgname=wally-cli
+version=2.0.0
+revision=1
+archs="x86_64"
```

Comment:
why is this limited to x86_64 (glibc)?

Reply:
I would've left it here for testing purposes when starting the file. I will update this to what works.

I cannot get these to cross compile and I've added what I can test

Reply:
after readin the cross build log,
```
aarch64-linux-musl-pkg-config: exec: "aarch64-linux-musl-pkg-config": executable file not found in $PATH
```
it's obvious you have to  `hostmakedepends="pkg-config"`
this makes it cross compile.
please remove the arch restriction if it's not necessary